src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -88,25 +87,21 @@ public static void Log(ILogger logger, IEnumerable<RestoreSummary> restoreSummar
                 // Display the errors summary
                 foreach (var restoreSummary in restoreSummaries)
                 {
-                    var errors = restoreSummary
-                        .Errors
-                        .Where(m => m.Level == LogLevel.Error)
-                        .ToList();
-
-                    if (errors.Count == 0)
-                    {
-                        continue;
-                    }
-
-                    logger.LogError(string.Empty);
-                    logger.LogError(string.Format(CultureInfo.CurrentCulture, Strings.Log_ErrorSummary, restoreSummary.InputPath));
-                    foreach (var error in errors)
-                    {
-                        foreach (var line in IndentLines(error.FormatWithCode()))
-                        {
-                            logger.LogError(line);
-                        }
-                    }
+                    // log warnings
```

Comment:
From the code it looks like it was intentional to skip writing warnings at the end in restore summary which could be fairly long for a large solution. Besides it will also have ripple effect of playing all those warnings over n over again in each restore summary. Even other systems like MSBuild doesn't list all the warnings at the end in build summary. So I think we should stick with `Errors` unless there is a strong reasoning of doing this.

Reply:
I see. I will remove the warning changes.